$NetBSD: patch-man_inadyn.conf.5,v 1.1 2015/04/04 07:59:34 snj Exp $

Use a more sensible cache-dir.  /var will be replaced by VARBASE.

--- man/inadyn.conf.5.orig	2015-04-03 22:40:32.000000000 -0700
+++ man/inadyn.conf.5	2015-04-03 22:41:17.000000000 -0700
@@ -51,7 +51,7 @@ startup-delay  60
 .br
 # pidfile      /var/run/inadyn.pid
 .br
-cache-dir      /etc/inadyn
+cache-dir      /var/db/inadyn
 .Pp
 # Primary account, used for most operations
 .br
